federated_content_connector/management/commands/tests/mock_responses.py

```diff
@@ -0,0 +1,361 @@
+                    "estimated_hours": 0,
+                    "content_language_search_facet_name": None,
+                    "enterprise_subscription_inclusion": False
+                }
+            ],
+            "entitlements": [
+                {
+                    "mode": "verified",
+                    "price": "149.00",
+                    "currency": "USD",
+                    "sku": "9023CB5",
+                    "expires": None
+                }
+            ],
+            "owners": [
+                {
+                    "uuid": "2906c944-5698-4e5d-bc18-a2bfe8d343af",
+                    "key": "edX",
+                    "name": "edX",
+                    "auto_generate_course_run_keys": True,
+                    "certificate_logo_image_url": None,
+                    "logo_image_url": None,
+                    "organization_hex_color": None,
+                    "description": None,
+                    "description_es": "",
+                    "homepage_url": None,
+                    "tags": [],
+                    "marketing_url": "school/edx",
+                    "slug": "edx",
+                    "banner_image_url": None,
+                    "enterprise_subscription_inclusion": False
+                }
+            ],
+            "image": None,
+            "short_description": "An introduction to the intellectual enterprises of computer science.",
+            "type": "97423c19-8254-43b6-bb7e-33b1f3efe246",
+            "url_slug": "introduction-to-computer-science",
+            "course_type": "executive-education-2u",
+            "enterprise_subscription_inclusion": False,
+            "excluded_from_seo": False,
+            "excluded_from_search": False,
+            "full_description": "This is CS50x , Harvard University's introduction to the ia.",
+            "level_type": None,
+            "subjects": [],
+            "prerequisites": [],
+            "prerequisites_raw": "",
+            "expected_learning_items": [],
+            "video": None,
+            "sponsors": [],
+            "modified": "2023-05-22T18:02:32.603508Z",
+            "marketing_url": None,
+            "syllabus_raw": "",
+            "outcome": "",
+            "original_image": None,
+            "card_image_url": None,
+            "canonical_course_run_key": "course-v1:edX+DemoX+Demo_Course",
+            "extra_description": None,
+            "additional_information": "",
+            "additional_metadata": {
+                "external_identifier": "",
+                "external_url": "http://www.edx.org",
+                "lead_capture_form_url": "",
+                "facts": [],
+                "certificate_info": None,
+                "organic_url": "",
+                "start_date": "2023-01-01T17:29:21Z",
+                "end_date": "2023-08-31T17:29:25Z",
+                "registration_deadline": "2023-05-31T17:29:42Z",
+                "variant_id": None,
```

Comment:
Should there be a `variant_id` defined if the course has `product_status: published` below? I believe all published Exec Ed courses have a `variant_id`.

Reply:
Updated the mock responses similar to prod data. Actually I have added these responses in devstack course-discovery that is why data is not good. It should be good now.